cdc/entry/mounter.go

```diff
@@ -488,7 +488,7 @@ func formatColVal(datum types.Datum, tp byte) (interface{}, error) {
 		return datum.GetMysqlDuration().String(), nil
 	case mysql.TypeJSON:
 		return datum.GetMysqlJSON().String(), nil
-	case mysql.TypeNewDecimal, mysql.TypeDecimal:
```

Comment:
should we report a fatal when meet a `mysql.TypeUnspecified `

Reply:
the default branch returns `datum.GetValue()`, I think it is ok if type is unspecified